TESTING.md

```diff
@@ -0,0 +1,96 @@
+# Snapcraft tests
+
+## Manual tests
+
+Snapcraft has a few manual tests documented in [manual-tests.md](manual-tests.md).
+These are tests for features that are not possible to automate, or that the complexity of automating them is too high while the time it takes to run them by hand is not too much.
+
+We try very hard to automate as much as possible and to keep the manual test suite as small as possible. Please do not add a manual test before discussing it with the team, and make sure that there are very good reasons not to automate it.
+
+### Staging server
+
+Snapcraft has the ability to upload snaps for publication in the Snappy Store. If you're working on a feature that requires you to interact with the store, you might want to use the staging server instead of the production store. To do
```

Comment:
This was the only paragraph with wrapping like that in HACKING.md. I was just trying to make it consistent here. I'm -1 on moving it to a separate PR.